pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -133,6 +133,7 @@ type ConverterContext struct {
 	FreePageReporting     bool
 	BochsForEFIGuests     bool
 	SerialConsoleLog      bool
+	DomainAttachment      *cmdv1.InterfaceDomainAttachment
```

Comment:
The name is to generic, consider something a bit more specific.

Reply:
Changed to `DomainAttachmentByInterfaceName`.